### ISSUE TITLE
Can't Import loguru (error un time.struct_time)

### ISSUE THREAD

**Issue by heliomcp** [Delgan/loguru#33]
```bash
$ python
Python 3.5.2 (v3.5.2:4def2a2901a5, Jun 25 2016, 22:01:18) [MSC v.1900 32 bit (Intel)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> from loguru import logger
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Program Files (x86)\Python35-32\lib\site-packages\loguru\__init__.py", line 10, in <module>
    from ._logger import Logger as _Logger
  File "C:\Program Files (x86)\Python35-32\lib\site-packages\loguru\_logger.py", line 30, in <module>
    start_time = now()
  File "C:\Program Files (x86)\Python35-32\lib\site-packages\loguru\_datetime.py", line 78, in now
    tzinfo = timezone(timedelta(seconds=local.tm_gmtoff), local.tm_zone)
AttributeError: 'time.struct_time' object has no attribute 'tm_gmtoff'
```

**Comment by Delgan** [comment]
Oh, thanks for the report!

I did not realize that `tm_gmtoff` was missing on some platform prior to Python 3.6, this is explicitly stated [in the documentation](https://docs.python.org/3/library/time.html).

I will try to find a workaround and fix it as soon as possible.

**Comment by Delgan** [comment]
Fixed and will be available in the next `v0.2.4` release, thanks again. 👍
